cmd/svcat/OWNERS

```diff
@@ -0,0 +1,8 @@
+approvers:
+- carolynvs
+reviewers:
+- eriknelson
+- jeremyrickard
+- kikisdeliveryservice
```

Comment:
Is @kikisdeliveryservice a member of the incubator org yet? If not, Kirsten, ping me and I can help you get added.

Reply:
I have no idea how org interaction works. I'm hoping we can be fairly liberal with adding to the reviewers section, and that it won't break anything if the person is not part of the org...

Reply:
👍

Reply:
@carolynvs I'm actually not a member of the org either, dunno if I need to be for the purposes of this or not.

Reply:
@eriknelson you should apply for membership too.